change/@uifabric-date-time-2019-07-17-11-47-47-date-time-readme.json

```diff
@@ -0,0 +1,8 @@
+{
+  "comment": "updated date-time readme now that it has major released",
+  "type": "patch",
```

Comment:
Are we patch change when updating things that don’t touch code like docs? Shouldn’t it be `none`?

Reply:
Not sure if beach ball gives a none option.

Reply:
It does…
![Screen Shot 2019-07-17 at 12 35 26](https://user-images.githubusercontent.com/29514833/61405766-777d4980-a88f-11e9-9327-eff031277e6e.png)

Reply:
```suggestion
  "type": "none",
```

Reply:
why is none between minor and major :-p